ethcore/src/lib.rs

```diff
@@ -97,7 +95,6 @@ extern crate rand;
 extern crate rayon;
 extern crate rlp;
 extern crate parity_util_mem;
-extern crate parity_util_mem as mem;
```

Comment:
Not sure this is correct, I think it is needed. cc @cheme

Reply:
Those are related to the malloc_size_of_derive macro, but if CI pass it is fine to remove it.